Sources/Purchasing/ProductsManager.swift

```diff
@@ -71,6 +71,17 @@ class ProductsManager: NSObject {
         productsFetcherSK1.cacheProduct(product)
     }
 
+    func clearCachedProducts() {
+        if #available(iOS 15.0, tvOS 15.0, watchOS 8.0, macOS 12.0, *),
+           self.systemInfo.storeKit2Setting == .enabledForCompatibleDevices {
+            Task {
```

Comment:
See `SwiftStyleGuide`, this should be `Task<Void, Never>` to make sure we don't ignore errors.

Reply:
👍🏼 I've to check again the style guide to be up to date.

![](https://media.giphy.com/media/vX9WcCiWwUF7G/giphy-downsized.gif)

Reply:
Done ✅ [a4f3e18](https://github.com/RevenueCat/purchases-ios/pull/1583/commits/a4f3e18e62042f9eaf98cdffac66a43e3aa2fbb7)